src/modules/navigator/mission_block.h

```diff
@@ -65,8 +65,27 @@ class MissionBlock : public NavigatorMode
 	MissionBlock(const MissionBlock &) = delete;
 	MissionBlock &operator=(const MissionBlock &) = delete;
 
+	/**
+	 * Check if the mission item contains a navigation position
+	 *
+	 * @return false if mission item should not be part of the trajectory
```

Comment:
I would not hint at the usage of this return value but only describe what it is. The user of the API should decide what to make of it.